client/src/components/UI_DateTime.re

```diff
@@ -19,22 +19,12 @@ module Transformer: {
   };
 };
 
-let component = ReasonReact.statelessComponent("DateDisplay");
-
-let make =
-    (
-      ~date: Js.Json.t,
-      ~transformer=Transformer.relative,
-      ~className=?,
-      _children,
-    ) => {
-  ...component,
-  render: _self => {
-    let date = date |> Js.Json.decodeString;
-    switch (date) {
-    | None => ReasonReact.null
-    | Some(date) =>
-      <time dateTime=date ?className> {date->transformer->str} </time>
-    };
-  },
+[@react.component]
+let make = (~date: Js.Json.t, ~transformer=Transformer.relative, ~className=?) => {
+  let date = date |> Js.Json.decodeString;
+  switch (date) {
+  | None => ReasonReact.null
```

Comment:
We can use `React.null` now.

Reply:
Replaced `ReasonReact.null` to `React.null`